src/app/sb-admin/components/pages/sb-organization/add-or-edit-org/add-or-edit-org.component.ts

```diff
@@ -5,129 +5,160 @@ import { OrganizationListService } from 'src/app/sb-admin/service/organization-l
 import { Message } from 'primeng/api';
 import { I18NextPipe } from 'angular-i18next';
 
-
 @Component({
-  selector: 'app-add-or-edit-org',
-  templateUrl: './add-or-edit-org.component.html',
-  styleUrls: ['./add-or-edit-org.component.scss']
+	selector: 'app-add-or-edit-org',
+	templateUrl: './add-or-edit-org.component.html',
+	styleUrls: ['./add-or-edit-org.component.scss'],
 })
 export class AddOrEditOrgComponent {
+	data: any;
+	mode!: 'Add' | 'Edit';
+	organization: any;
+	addOrgForm!: FormGroup;
+	EditOrgForm!: FormGroup;
+	submitted: boolean = false;
+	messages!: Message[];
+	isDisabled: boolean = true;
+	currentOrgValue!: string;
+	CurrentChannelName!: string;
 
-  data: any;
-  mode!: 'Add' | 'Edit';
-  organization: any;
-  addOrgForm!: FormGroup;
-  EditOrgForm!: FormGroup
-  submitted: boolean = false;
-  messages!: Message[];
-  isDisabled: boolean = true;
-  currentOrgValue!: string;
-  CurrentChannelName!: string;
-
-  constructor(public formBuilder: FormBuilder,
-    public ref: DynamicDialogRef,
-    public dialogConfig: DynamicDialogConfig,
-    private organizationListService: OrganizationListService,
-    private i18nextPipe: I18NextPipe) {
-    this.data = this.dialogConfig.data;
-    this.mode = this.data.mode;
-    this.organization = this.data.organization;
-  }
+	constructor(
+		public formBuilder: FormBuilder,
+		public ref: DynamicDialogRef,
+		public dialogConfig: DynamicDialogConfig,
+		private organizationListService: OrganizationListService,
+		private i18nextPipe: I18NextPipe,
+	) {
+		this.data = this.dialogConfig.data;
+		this.mode = this.data.mode;
+		this.organization = this.data.organization;
+	}
 
-  ngOnInit() {
-    this.initializeAddForm();
-    this.initialzeEditForm();
-  }
```

Comment:
as discussed , please undo changes and remove the changes which are caused by code prettier